tools/terraform_sync_tool/qa/terragrunt.hcl

```diff
@@ -0,0 +1,43 @@
+# Indicate where to source the terraform module from.
+# The URL used here is a shorthand for
+# "tfr://registry.terraform.io/terraform-aws-modules/vpc/aws?version=3.5.0".
+# Note the extra `/` after the protocol is required for the shorthand
+# notation.
+
+locals {
+  #TODO: Update your GCP Project ID
+  gcp_project_id = "YOUR_GCP_PROJECT_ID" 
+}
+
+inputs = {
+  project_id = local.gcp_project_id
+  gcp_region = "us-central1"
+}
+
+generate "provider" {
+  path      = "provider.tf"
+  if_exists = "overwrite"
+  contents  = <<EOF
+provider "google" {
+  project = "${local.gcp_project_id}"
+}
+  EOF
+}
+
+remote_state {
+  backend = "gcs"
+  config = {
+    project  = local.gcp_project_id
+    location = "us"
+    bucket   = "synctooltest"
```

Comment:
Have the user provide their own bucket name because bucket names must be unique across all GCP

Reply:
Addressed in [Use provided bucket name](https://github.com/GoogleCloudPlatform/bigquery-utils/pull/290/commits/f699fdd13effb398e513436df2ef901f45f63569)